master/buildbot/changes/gerritchangesource.py

```diff
@@ -22,6 +22,9 @@
 from twisted.internet.protocol import ProcessProtocol
 from twisted.python import log
 
+def defaultProjectNameTranslator(gerritProjectName):
+    """Just return the last part of the project if it contains slashes"""
+    return gerritProjectName.rpartition('/')[-1]
```

Comment:
should we change default behaviour? Or put this as an optional basenameProjectNameTranslator()